applications/ray/kuberay-tpu-webhook/deployments/mutate-pod-webhook-cfg.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: admissionregistration.k8s.io/v1
 kind: MutatingWebhookConfiguration
 metadata:
-  name: kuberay-tpu-webhook-cfg
+  name: kuberay-tpu-mutate-pod-cfg
   annotations:
     cert-manager.io/inject-ca-from: default/kuberay-tpu-webhook-certs
```

Comment:
The cert-manager reference here needs to be updated to `ray-system/kuberay-tpu-webhook-certs`

Reply:
Done.